composer.json

```diff
@@ -73,55 +73,65 @@
   },
   "require-dev": {
     "composer/composer": "^1.10.0",
```

Comment:
Where is this used?

Reply:
Don't know.  Probably unused I guess.

Reply:
Removed. It does not seem to make any sense.